src/utils/validation.ts

```diff
@@ -62,7 +77,9 @@ const _cardNumberValidator = (value: string) => {
 const _expirationDateValidator = expirationDate;
 
 // eslint-disable-next-line @typescript-eslint/default-param-last
-const _maskValidator = (mask: Mask = [], value: string) => {
+const _maskValidator = (options: ValidatorOptions = {}, value: string) => {
+  const mask = options?.mask ?? [];
```

Comment:
Did the default arg fail?

Reply:
You mean leaving `mask: Mask = []`? yeah that causes issues w/ the events